scraper.rb

```diff
@@ -35,13 +35,20 @@ def tidy
 end
 
 def scrape_list(url)
-  MembersList.new(response: Scraped::Request.new(url: url).response).members
+  MembersList.new(response: Scraped::Request.new(url: url).response)
+             .members
+             .map(&:to_h)
+             .map do |member|
+              member.merge(MemberPage.new(
```

Comment:
The indentation here doesn't seem quite right, I think you need an extra space (and then fix the two lines below as well.